doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,298 @@
+
+!listing lfr/heterogeneous_single_assembly_3D/cross_section/Step2/Fuel/LFR_127Pin_Fuel_1D_9g.mcc3.sh start=&material end=EOF include-end=False
+
+The superposition rule is depicted in [MCCgenProc]. The first two boxes represent [#xs1st] to generate a TWODANT spectrum and the third up and low two boxes represent [#xs2nd] for non-fuel and fuel regions, respectively. If `c_geometry_type=mixture`, the 0D UFG solution of mixture is adjusted by the ratio of the TWODANT spectrum to the 0D solution condensed to the TWODANT group structure. If the TWODANT group structure is UFG, just the TWODANT spectrum is used as a weighting function for condensation. If `c_geometry_type` is not `mixture`, each region-wise UFG solution is adjusted by the ratio of the TWODANT spectrum to the whole domain-averaged solution condensed to the TWODANT group structure. To simply state, the adjusted region-wise UFG solution is obtained by multiplying the spatial self-shielding factor obtained by the MC2-3 1D calculation to the TWODANT spectrum.
+
+!media lfr/MCCgenProc.jpg
+       style=width:100%
+       id=MCCgenProc
+       caption=MC2-3 cross section generation procedure with the axial leakage effect considered using TWODANT
+
+Without `l_spatial_homogenization=F` in an input, which is easy to make mistake, one will end up using the assembly average cross section for each region in heterogeneous fine-mesh transport calculation. [XS_Hom_Vs_Reg] shows the relative difference of assembly average cross section to hexagonal ring-wise fuel cross section for U-238 capture. These differences are basically the same as the spatial self-shielding factor of each ring: the ratio of hexagonal ring-wise flux to assembly average flux. Since spectrum is softer for peripheral regions than for central regions, central region flux is lower than the assembly average flux at low energy and higher at high energy. Thus, a test showed that the use of assembly average cross section instead of region-wise cross section resulted in about +400 pcm error in k-effective due to underestimation of low energy capture and overestimation of high energy fission.
```

Comment:
```suggestion
Without `l_spatial_homogenization=F` in an input, which is an easy mistake to make, one will end up using the assembly average cross section for each region in heterogeneous fine-mesh transport calculation. [XS_Hom_Vs_Reg] shows the relative difference of assembly average cross section to hexagonal ring-wise fuel cross section for U-238 capture. These differences are basically the same as the spatial self-shielding factor of each ring: the ratio of hexagonal ring-wise flux to assembly average flux. Since spectrum is softer for peripheral regions than for central regions, central region flux is lower than the assembly average flux at low energy and higher at high energy. Thus, a test showed that the use of assembly average cross section instead of region-wise cross section resulted in about +400 pcm error in k-effective due to underestimation of low energy capture and overestimation of high energy fission.
```